back/routes/bookings.js

```diff
@@ -120,9 +120,10 @@ router.post('/', (request, response) => {
 });
 
 
-router.delete('/tourist/:id', (req, res) => {
+router.delete('/tourist/:id/:id', (req, res) => {
   const idVacationer = req.params.id;
-  connection.query('DELETE FROM booking WHERE tourist_id = ?', [idVacationer], err => {
+  const idEvent = req.params.id;
```

Comment:
I think req.params.id and req.params.id are the same, give them different names in route path